frontend/src/concepts/pipelines/content/createRun/utils.ts

```diff
@@ -28,7 +28,7 @@ const isValidDate = (value?: RunDateTime): boolean => {
   }
 
   const date = new Date(`${value.date} ${value.time}`);
-  return date.toString() !== 'Invalid Date';
+  return date.toISOString() !== 'Invalid Date';
```

Comment:
Revert this change.
`date.toISOString()` does not return `Invalid Date` but instead throws an error.